### PR TITLE
[Migrate Simple Payments] Remove MIGRATION_SIMPLE_PAYMENTS FF with unused code

### LINKED ISSUE #11266: [Migrate Simple Payments] Remove FF And unused code

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #11266 

### Description

The PR removes MIGRATION_SIMPLE_PAYMENTS FF with unused code. Note that we can not remove all the simple payment features, as we reuse this for "Try Tap To Pay" functionality.

### Testing instructions

Not sure what can be tested here

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is larger than 300 lines of changes. Please consider splitting it into smaller PRs for easier and faster reviews.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/205">18.2</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false"><p>This PR contains changes to Tracks-related logic. Please ensure (<strong>author and reviewer</strong>) the following are completed:</p>

<ul>
  <li>The tracks events must be validated in the Tracks system.</li>
  <li>Verify the internal Tracks spreadsheet has also been updated.</li>
  <li>Please consider registering any new events.</li>
  <li>The PR must be assigned the <strong>category: tracks</strong> label.</li>
</ul></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 *** PLEASE FOLLOW THIS FORMAT: [<priority indicator, more stars = higher priority>] <description> [<PR URL>]
 18.2
 -----
-
+- [**] Payments: We sunsetting simple payments in favour of order creation flow with custom amount set [https://github.com/woocommerce/woocommerce-android/pull/11274]
```
Comment (backwardstruck) [inline RELEASE-NOTES.txt]: Minor nitpick, should it read?

```
- [**] Payments: We are sunsetting simple payments in favour of the order creation flow with custom amount set [https://github.com/woocommerce/woocommerce-android/pull/11274]
```

**Review by backwardstruck — APPROVED** [review]
Just one non-blocking comment. Feel free to merge.

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (kidinov) [reply RELEASE-NOTES.txt]: Good catch. Fixed here - b606399e6bbcc5bdc39c543a9d5b53eb4932cc7b

**Pull request merged**